sources/packages/forms/src/form-definitions/studentprofile.json

```diff
@@ -3711,92 +3711,33 @@
           "tags": [],
           "properties": {},
           "conditional": {
-            "show": false,
-            "when": "mode",
-            "eq": "aest-account-approval",
-            "json": ""
-          },
-          "customConditional": "",
-          "logic": [],
-          "attributes": {},
-          "overlay": {
-            "style": "",
-            "page": "",
-            "left": "",
-            "top": "",
-            "width": "",
-            "height": ""
-          },
-          "type": "checkbox",
-          "name": "",
-          "value": "",
-          "input": true,
-          "placeholder": "",
-          "prefix": "",
-          "suffix": "",
-          "multiple": false,
-          "unique": false,
-          "refreshOn": "",
-          "dataGridLabel": true,
-          "labelPosition": "right",
-          "widget": null,
-          "validateOn": "change",
-          "showCharCount": false,
-          "showWordCount": false,
-          "allowMultipleMasks": false,
-          "addons": [],
-          "id": "exdbzkf"
-        },
-        {
-          "label": "I Agree to the terms and conditions of the StudentAid BC Declaration",
-          "labelWidth": "",
-          "labelMargin": "",
-          "description": "",
-          "tooltip": "",
-          "shortcut": "",
-          "inputType": "checkbox",
-          "customClass": "",
-          "tabindex": "",
-          "hidden": false,
-          "hideLabel": false,
-          "autofocus": false,
-          "disabled": true,
-          "tableView": false,
-          "modalEdit": false,
-          "defaultValue": false,
-          "persistent": "client-only",
-          "protected": false,
-          "dbIndex": false,
-          "encrypted": false,
-          "redrawOn": "",
-          "clearOnHide": true,
-          "customDefaultValue": "",
-          "calculateValue": "value=data.sinConsent;",
-          "calculateServer": false,
-          "allowCalculateOverride": false,
-          "validate": {
-            "required": false,
-            "customMessage": "",
-            "custom": "",
-            "customPrivate": false,
-            "json": "",
-            "strictDateValidation": false,
-            "multiple": false,
-            "unique": false
-          },
-          "errorLabel": "",
-          "errors": "",
-          "key": "sinConsentChecked",
-          "tags": [],
-          "properties": {},
-          "conditional": {
-            "show": true,
-            "when": "mode",
+            "show": "",
+            "when": "",
             "eq": "aest-account-approval",
             "json": ""
           },
           "customConditional": "",
-          "logic": [],
+          "logic": [
+            {
+              "name": "Disable checkbox for Client AEST",
+              "trigger": {
+                "type": "javascript",
+                "javascript": "result = (data.mode && data.mode === 'aest-account-approval');"
```

Comment:
like other places we can add the property clientType here also to be in sync. 

Also this property which is used for logic must be added to the Model at Vue side with comment for clarity.

Reply:
Also I am missing to see this mode populated at vue side.

Reply:
@dheepak-aot there is no reason to add the clientType here. What would be the benefit?
Als the property mode was already there, it is not part to these changes.
![image](https://user-images.githubusercontent.com/61259237/192030993-61b3e878-2781-4b25-86d7-a2beb8fabf67.png)

Reply:
I got the know that there is an existing property and we can re-use it from @henryoforeh-aot. I am good.